frontend/src/components/BackButton.tsx

```diff
@@ -0,0 +1,25 @@
+import BackIcon from "../assets/backIcon.png";
+
+const BackButton = () => {
+  return (
+    <button
+      style={{
+        width: "45px",
+        height: "45px",
+        display: "inline-flex",
+        alignItems: "center",
+        justifyContent: "center",
+      }}
```

Comment:
was there a reason why you didn't use tailwindcss? much prefer to stick with the tailwind for consistency in code

Reply:
many apologies, I may have forgotten